web_src/js/components/RepoActionView.vue

```diff
@@ -526,12 +594,19 @@ export function ansiLogToHTML(line) {
 
 .job-step-section .job-step-logs {
   font-family: monospace, monospace;
+  margin: 8px 0px;
+  font-size: 12px;
 }
 
 .job-step-section .job-step-logs .job-log-line {
   display: flex;
 }
 
+.job-step-section .job-step-logs .job-log-line:hover {
+  color: var(--color-console-fg);
+  background-color: var(--color-black-light);
```

Comment:
Better introduce a `--color-console-hover` with alpha value like `#ffffff10`.

Reply:
Modified in https://github.com/go-gitea/gitea/pull/24718/commits/05330a1f697e47ecd20bea869a1225124229a7ce